dev/io.openliberty.jcache.internal/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,61 @@
+###############################################################################
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# -------------------------------------------------------------------------------------------------
+#CMVCPATHNAME io.openliberty.jcache.internal/resources/OSGI-INF/l10n/metatype.properties
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+
+
+#
+# jCachingProvider
+#
+jCachingProvider=JCache CachingProvider
+jCachingProvider.desc=Configuration for the JCache CachingProvider
+
+providerClass=JCache CachingProvider class
+providerClass.desc=The fully-qualified class name of the JCache javax.cache.CachingProvider to use.
+
+libraryRef=Library references
+libraryRef.desc=A comma-separated list of libraries that will used to access the JCache implementation and any custom classes that may be stored in the cache.  
```

Comment:
A comma-separated list of libraries that are used to access the JCache implementation and any custom classes that might be stored in the cache.

Reply:
Updated.